pkg/docker/cockpit-atomic-storage

```diff
@@ -337,7 +333,7 @@ def drive_class(d):
 
 can_manage = None
 
-def get_info():
+def get_info(got_uevent):
```

Comment:
how about I add a comment while merging:
`got_uevent` is currently unused, but mvollmer wants to use it sometime in the future